### PR TITLE
Bw 546

### PR DESCRIPTION
**Pull request by milansar**

### CODE REVIEW

File: projects/trivia/src/app/user/components/profile-settings/profile-settings.ts [hunk 1]
```diff
@@ -45,68 +50,142 @@ export class ProfileSettings {
...
 
         this.toggleLoader(true);
 
         this.fb = formBuilder;
 
+
+
+        this.tagsObs = this.store.select(getTags);
+        this.subscriptions.push(this.tagsObs.subscribe(tagsAutoComplete => this.tagsAutoComplete = tagsAutoComplete));
+
```
Comment (anihalaney) [inline projects/trivia/src/app/user/components/profile-settings/profile-settings.ts]: Nested subscriptions?

File: projects/trivia/src/app/user/components/profile-settings/profile-settings.ts [hunk 2]
```diff
@@ -45,68 +50,142 @@ export class ProfileSettings {
...
+                        this.initializeOtherUserProfile();
+                    }
+                });
+            }
+        });
+
+    }
+
+    initData() {
+        this.userObs = this.store.select(appState.coreState).pipe(select(s => s.user));
```
Comment (anihalaney) [inline projects/trivia/src/app/user/components/profile-settings/profile-settings.ts]: Nested subscriptions?

File: projects/trivia/src/app/user/components/profile-settings/profile-settings.ts [hunk 3]
```diff
@@ -45,68 +50,142 @@ export class ProfileSettings {
...
 
-                if (user.roles && user.roles['bulkuploader']) {
-                    this.user.bulkUploadPermissionStatus = profileSettingsConstants.APPROVED;
-                }
+    initializeOtherUserProfile() {
+        this.categoryDictObs = this.store.select(categoryDictionary);
+        this.subscriptions.push(this.categoryDictObs.subscribe(categoryDict => this.categoryDict = categoryDict));
 
-                this.toggleLoader(false);
+        this.userDict$ = this.store.select(appState.coreState).pipe(select(s => s.userDict));
```
Comment (anihalaney) [inline projects/trivia/src/app/user/components/profile-settings/profile-settings.ts]: Nested subscriptions?

**Comment by milansar** [comment]
Closing this pull request as this would be superseded by another pull request from BW-549

**Pull request closed without merging**